### ISSUE TITLE
Enum

### ISSUE THREAD

**Issue by benman1** [python/mypy#8752]
I am trying the enum examples from other issues or the [blog post introducing enums](http://mypy-lang.blogspot.com/2017/), but I can't get them to work. I am not sure what's wrong - I compiled other functions (without enums) before, and all was fine, but enums give me all kinds of errors.

My examples look like this:
```python
from enum import IntEnum

Color = IntEnum('Color', ['red', 'blue', 'green'])
Color.red
```

This is the error for this example:
```bash
running build_ext
building 'enum_example' extension
gcc -pthread -B /home/ben/anaconda3/compiler_compat -Wl,--sysroot=/ -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/ben/anaconda3/lib/python3.6/site-packages/mypyc/lib-rt -I/home/ben/anaconda3/include/python3.6m -c build/__native.c -o build/temp.linux-x86_64-3.6/build/__native.o -O3 -Werror -Wno-unused-function -Wno-unused-label -Wno-unreachable-code -Wno-unused-variable -Wno-unused-command-line-argument -Wno-unknown-warning-option -Wno-unused-but-set-variable
build/__native.c: In function ‘CPyDef___top_level__’:
build/__native.c:183:29: error: ‘CPyType_Color’ undeclared (first use in this function); did you mean ‘PyType_Slot’?
     cpy_r_r28 = (PyObject *)CPyType_Color;
                             ^~~~~~~~~~~~~
                             PyType_Slot
build/__native.c:183:29: note: each undeclared identifier is reported only once for each function it appears in
build/__native.c: At top level:
cc1: error: unrecognized command line option ‘-Wno-unknown-warning-option’ [-Werror]
cc1: error: unrecognized command line option ‘-Wno-unused-command-line-argument’ [-Werror]
cc1: all warnings being treated as errors
error: command 'gcc' failed with exit status 1
```

I have the latest (released) version of mypy and a recent version of gcc
```bash
mypy==0.770
mypy-extensions==0.4.3
gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
```

What I can compile and execute is this:
```python
from enum import Enum, auto

class ExpEnum(Enum):
    ONE = auto()
    TWO = auto()

value = ExpEnum['ONE']
```
However, I want to use members like 'ONE-TWO', so I can't use this style as far as I understand.

Ultimately, the functionality I want to get is:
```python
Colors = Enum('colors', ['red', 'green', 'blue'])
assert Colors['red'] == 1
assert Colors.red == 1
# or even better: 0; I've noticed the 'start' option doesn't work
```

If I am doing something stupid, please tell me. I am just trying to make this work.

**Comment by JelleZijlstra** [comment]
I'm guessing mypyc just hasn't added support for this feature yet. The examples should work if you just use mypy (not mypyc).

**Comment by benman1** [comment]
Thanks! That makes sense.

**Comment by JukkaL** [comment]
@benman1 Can you file this again  as a mypyc issue (https://github.com/mypyc/mypyc/issues)?

**Comment by benman1** [comment]
Will do!

**Comment by benman1** [comment]
https://github.com/mypyc/mypyc/issues/721
